frontend/www/js/omegaup/components/course/AssignmentList.vue

```diff
@@ -168,14 +182,31 @@ import { Vue, Component, Prop } from 'vue-property-decorator';
 import { omegaup } from '../../omegaup';
 import T from '../../lang';
 
-@Component
+import {
+  FontAwesomeIcon,
+  FontAwesomeLayers,
+  FontAwesomeLayersText,
+} from '@fortawesome/vue-fontawesome';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+import { library } from '@fortawesome/fontawesome-svg-core';
+library.add(fas);
+
+@Component({
+  components: {
+    'font-awesome-icon': FontAwesomeIcon,
+    'font-awesome-layers': FontAwesomeLayers,
+    'font-awesome-layers-text': FontAwesomeLayersText,
+  },
+})
 export default class CourseAssignmentList extends Vue {
   @Prop() assignments!: omegaup.Assignment[];
   @Prop() courseAlias!: string;
+  @Prop() visibilityMode!: omegaup.VisibilityMode;
 
   testsOrderChanged = false;
   homeworksOrderChanged = false;
   T = T;
+  omegaup = omegaup;
```

Comment:
nit:

```suggestion
  VisibilityMode = omegaup.VisibilityMode;
```

esto es lo único que se utiliza de `omegaup`.

Reply:
Listo